packages/rest/CHANGELOG.md

```diff
@@ -24,13 +24,14 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### BREAKING CHANGES
 
+* **rest:** Message printed by REST LogError action changed from
+`Unhandled error` to `Request failed`. Message pattern rules
+(e.g. alerting rules) based on this pattern need to be updated accordingly.
```

Comment:
I'm not sure if the `lerna` release process will override the commit in the future.

Reply:
I hope it will not, is there a way how to try it out?

Reply:
I guess we could just merge it and re-recheck it after the next publish

Reply:
We can run `npx lerna version --no-push` to try out.